### DISCUSSION TITLE
Roadmap Discussion

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [nebari-dev/nebari#198] [Ideas]
@dharhas several weeks ago you mentioned that you guys would like to have a Qhub roadmap discussion with me and @jkellndorfer.   

We are both jamming on Qhub and are still very interested!  😄 

If you are willing, it would be great to schedule a call so we could meet the team (at least virtually!), but we could also just discuss here if you prefer.

**Comment by jkellndorfer** [comment]
yes, I'd be up for that 👍

**Comment by tylerpotts** [comment]
@jkellndorfer @rsignell-usgs I also think that would be a great idea! Let me sync with the team and get back to you both

**Comment by dharhas** [comment]
Yup. Let me see if I can find a time next week before folks disappear for Christmas

**Comment by dharhas** [comment]
Let's see if we can nail down a time this week.

https://doodle.com/poll/hz8pwvsa49skptrf

@jkellndorfer @rsignell-usgs

**Comment by dharhas** [comment]
So Wednesday at 2-3pm central or Thursday 1-3pm if we think we want more than an hour. Any preference?

**Reply by costrouc** [reply]
I would prefer Thursday 1-3 PM. But fine if that time works better for others. I have a hard stop at 3 PM ET on Wednesday.

**Comment by jkellndorfer** [comment]
Thursday seems preferable to me as well as we are less pressured.

Josef

**Comment by dharhas** [comment]
Ok Thursday 1-3pm it is. I have a google meet setup. We will use this [google meet link](https://meet.google.com/gun-zaae-oij) for the call. Not sure of a good way to send y'all a calendar invite unless you want to send me your email addresses.

Aah you can click on this to add to your calendar.

<a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=NDc2MHBoNG51bGExbmhpaWdicTJjajQwMHIgZGhhcmhhc0BxdWFuc2lnaHQuY29t&amp;tmsrc=dharhas%40quansight.com"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>

**Comment by jkellndorfer** [comment]
Great. I am all set copying the link to my calendar.
Look forward to talking to you all on Thursday.

Josef

**Comment by laisbsc** [comment]
Hello folks,

Hope you're all well :)  We listed some ideas for our _get together_, you can have a look at them in [here](https://hackmd.io/@quansight/B15z6II2w).

Really looking forward for the discussion  on Thursday!

Chat soon! 🚀

**Reply by dharhas** [reply]
@rsignell-usgs @jkellndorfer please feel free to add ideas to the document.

**Comment by rsignell-usgs** [comment]
Guessing 1-3pm today Central, not Eastern?

**Comment by rsignell-usgs** [comment]
@costrouc @dharhas @tonyfast @laisbsc @jkellndorfer @tylerpotts @brl0 
Great meeting yesterday!     It was awesome to meet the team, and I'm even more excited about Qhub than I was before, which I didn't think was possible!  😄 

Sidenote: I installed k9s on Windows (yes, I'm a US Gov employee) using Scoop and I'm loving it!!!

**Comment by jkellndorfer** [comment]
@costrouc <https://github.com/costrouc> @dharhas <https://github.com/dharhas> @tonyfast <https://github.com/tonyfast> @laisbsc <https://github.com/laisbsc> @rsignell-usgs @tylerpotts <https://github.com/tylerpotts> @brl0 <https://github.com/brl0>Yes, same comment from me. Really great to meet everyone and continue with the excitement of qhub! 
Look forward to next steps.

Cheers,
Joef

**Reply by laisbsc** [reply]
Awesome to catch up too, @jkellndorfer! Hope you enjoyed the holidays.

I was browsing around and thought either you or @rsignell-usgs could be interested in submitting something for [GeoPython 2021](https://2021.geopython.net/).

We could maybe even think about hosting a workshop if you fancy. 😄  For now, just food for thought.

But please let me know what you think!

Chat soon!

**Reply by dharhas** [reply]
huh, I'm surprised that I've never heard of this conference before... ah looks like it started in 2016 and is mainly European until it went online recently, I guess that is why it never crossed my radar.

**Reply by rsignell-usgs** [reply]
I've never heard of this either, which makes me even more curious to attend.  😄 
I think a workshop showing folks how to set up their own qhub would be great!

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
